packages/blade/src/components/Button/_decisions/decisions.md

```diff
@@ -0,0 +1,68 @@
+# Button Decisions <!-- omit in toc -->
+- [Design](#design)
+- [Internal BaseButton Component](#internal-basebutton-component)
+  - [BaseButton API](#basebutton-api)
+- [Button Component](#button-component)
+  - [Button API](#button-api)
+- [Component Breakdown](#component-breakdown)
+- [Open Questions](#open-questions)
+
+## Design
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=5200%3A0) to all variants of the Button component
+
+## Internal BaseButton Component
+Internal component that exposes certain extra props to enable creation of Component Level Action Buttons like `positive`, `negative`, `neutral`
+
+### BaseButton API
+
+| Prop | Type | Required | Default | Description |
+|---|---|---|---|---|
+| variant | `primary`,  `secondary`,  `tertiary`,  `link` | No | `primary` | - The variant of the button to be rendered.<br><br>Note: `link` will look like an inline Link component but will be rendered as a `button` element |
+| intent | `none`, `positive`,`negative`, `notice`, `info`, `neutral` | No | `none` | The intent of the button to be rendered.<br><br>  Note: <br> - We will use this to set the intent for internal Blade components that have positive, negative, notice, info, or neutral buttons like an Alert component<br> - All variants of the button will be using the `none` intent which would indicate there is no effect added by the `intent` prop |
```

Comment:
@chaitanyadeorukhkar change the values for `intent` here?

Reply:
Done